x-pack/plugins/cloud_security_posture/public/pages/rules/rules_table.tsx

```diff
@@ -124,12 +136,36 @@ const getColumns = ({
     field: 'updatedAt',
     name: TEXT.UPDATED_AT,
     width: '15%',
-    render: timestampRenderer,
+    render: (timestamp) => moment.duration(moment().diff(timestamp)).humanize(),
   },
   {
     field: 'attributes.enabled',
     name: TEXT.ENABLED,
-    render: createRuleEnabledSwitchRenderer({ toggleRule }),
+    render: (enabled, rule) => (
+      <EuiToolTip
+        content={
+          enabled ? (
+            <FormattedMessage
+              id="xpack.csp.rules.deactivateRuleLabel"
```

Comment:
i think some sub-sections should be added, and the message type should be changed to `Tooltip`.
https://github.com/elastic/kibana/blob/main/packages/kbn-i18n/GUIDELINE.md#:~:text=it%27s%20a%20placeholder)%2C-,Tooltip,-(if%20it%27s%20a

something like `xpack.csp.rules.ruleTable.ruleTableColumn.deactivateRuleTooltip`